src/legacy/core_plugins/telemetry/public/hacks/telemetry_init.ts

```diff
@@ -30,8 +30,9 @@ function telemetryInit($injector: any) {
   const $http = $injector.get('$http');
 
   const telemetryEnabled = npStart.core.injectedMetadata.getInjectedVar('telemetryEnabled');
+  const usageFetcher = npStart.core.injectedMetadata.getInjectedVar('telemetryUsageFetcher');
 
-  if (telemetryEnabled) {
+  if (telemetryEnabled && usageFetcher === 'browser') {
```

Comment:
Just a thought - is it worth extracting the `browser` and `server` to an Enum?
It's just a good way of ensuring you get typing on the variables and avoid pitfalls like typos.

Definitely a nit and non blocker, but I'd love to see us bias towards Enums instead of strings whenever possible.